OCM/tf/ocm_tf_checker.cc

```diff
@@ -1058,7 +1107,7 @@ std::set<std::string> GetTFSupportedOPs(std::string device_id,
         ov_based_op_list[e.first].insert(e.second.begin(), e.second.end());
       }
     }
-    if (ov_version[0] > 2022 || ov_version[1] >= 1) {
+    if (ov_version[0] >= 2022 && ov_version[1] >= 1) {

```

Comment:
Please explain why we changed the qualifying rule to use && instead of ||

Reply:
In our scripts, we are taking 2022.2.0 into ov_version as [macro, micro, patch] i.e. [2022, 2, 0] 
Assume our goal is to enable a specific data type only if the ov version is equal to or greater than 2022.2.0

In "||" case, if (ov_version[0] > 2022 || ov_version[1] >= 1) {
If we are using 2021.4 here with ov_version[0], it will fail, but in the next condition, as 4>1 is satisfied and might pass, op support will be enabled for 2021.4 also, which should not happen.

In "&&" case, if (ov_version[0] >= 2022 && ov_version[1] >= 1) {
The data type support will be enabled only if the OV version is greater than 2022 and greater than 1. 